tests/test_unit.py

```diff
@@ -46,8 +58,56 @@ def test_interpreter_version(current_interpreter: Interpreter) -> None:
 
 
 def test_interpreter_version_info(current_interpreter: Interpreter) -> None:
-    assert current_interpreter.version_info() == (
-        sys.version_info[0],
-        sys.version_info[1],
-        sys.version_info[2],
-    )
+    assert current_interpreter.version_info() == sys.version_info[:3]
+
+
+def test_get_python_sitepackages() -> None:
+    assert get_python_sitepackages("python") == site.getsitepackages()
+
+
+@pytest.mark.skip(reason="We are unable to execute venv commands in tests")
+def test_get_venv_sitepackages(current_venv: Venv) -> None:
+    instances = list(current_venv.instances(default_venv_pattern))
+    assert instances
+    assert get_venv_sitepackages(instances[0].venv_path()) == site.getsitepackages()
+
+
+def test_interpreter_pythonpath(current_interpreter: Interpreter) -> None:
+    assert current_interpreter.pythonpath == ":".join(site.getsitepackages())
+
+
+def test_venv_instance_pythonpath(current_venv: Venv) -> None:
+    """Test the value of VenvInstance.pythonpath.
+
+    The result from VenvInstance.pythonpath
+      When no VenvInstance.pkgs are defined
+        Will be the Interpreter.pythonpath
+    """
+    instances = list(current_venv.instances(default_venv_pattern))
+    assert instances
+    for instance in instances:
+        assert instance.pythonpath == instance.py.pythonpath
+
+
+def test_venv_instance_with_pkgs_pythonpath(current_venv: Venv) -> None:
```

Comment:
I think I'd much rather have integration tests for this sort of thing with real usage (`riot run ...`). While it's great we can test at a high granularity, these kinds of tests are hard to maintain as they're brittle to many sources that we don't intend to test here (eg. we change the `get_venv_sitepackages` name or implementation).

I'd actually be more comfortable with a subprocess test that has riot construct a Venv and then checks to see which virtualenv the package got installed to.

WDYT?

Reply:
yeah, this makes sense. I wasn't thinking about integration tests right now, but probably good thing to figure out now since our test coverage in general is lacking, good to get ahead of it.